[Android] App crash at startup with Vulkan

**Describe the bug**
The Empty test app crashes at startup in Android with «Didn't find class "tech.kinc.KincActivity" on path: DexPathList» (without changing the package name of the app).

**To Reproduce**
Steps to reproduce the behavior:
1. Compile Empty test app to Android using `kmake android`, them build with Android Studio.

**Execution Environment:**
- Host system (where you compile your code): Linux Mint
- Target system (where you run your code): Android Emulator with Pixel 3a API 34.
- IDE and/or compiler used: Android Studio.
- Kinc revision: f7d6ef53
- Kinc build output: Using Kinc (f7d6ef53)
- Application output (if it runs):

LogCat message:

```
FATAL EXCEPTION: main
Process: tech.kinc, PID: 9817
java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{tech.kinc/tech.kinc.KincActivity}: java.lang.ClassNotFoundException: Didn't find class "tech.kinc.KincActivity" on path: DexPathList[[zip file "/data/app/~~bG0DUNK4hR-DglERMlG3Vg==/tech.kinc-_9dRiev2AfQleniAxFsP0w==/base.apk"],nativeLibraryDirectories=[/data/app/~~bG0DUNK4hR-DglERMlG3Vg==/tech.kinc-_9dRiev2AfQleniAxFsP0w==/lib/x86_64, /data/app/~~bG0DUNK4hR-DglERMlG3Vg==/tech.kinc-_9dRiev2AfQleniAxFsP0w==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3689)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8177)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: java.lang.ClassNotFoundException: Didn't find class "tech.kinc.KincActivity" on path: DexPathList[[zip file "/data/app/~~bG0DUNK4hR-DglERMlG3Vg==/tech.kinc-_9dRiev2AfQleniAxFsP0w==/base.apk"],nativeLibraryDirectories=[/data/app/~~bG0DUNK4hR-DglERMlG3Vg==/tech.kinc-_9dRiev2AfQleniAxFsP0w==/lib/x86_64, /data/app/~~bG0DUNK4hR-DglERMlG3Vg==/tech.kinc-_9dRiev2AfQleniAxFsP0w==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:95)
	at androidx.core.app.CoreComponentFactory.instantiateActivity(CoreComponentFactory.java:45)
	at android.app.Instrumentation.newActivity(Instrumentation.java:1378)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3676)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922) 
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103) 
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139) 
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96) 
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443) 
	at android.os.Handler.dispatchMessage(Handler.java:106) 
	at android.os.Looper.loopOnce(Looper.java:205) 
	at android.os.Looper.loop(Looper.java:294) 
	at android.app.ActivityThread.main(ActivityThread.java:8177) 
	at java.lang.reflect.Method.invoke(Native Method) 
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552) 
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971) 
```

Maybe there is something missing in my system?

--- a/Sources/kinc/log.h
+++ b/Sources/kinc/log.h
@@ -86,6 +86,22 @@ void kinc_log(kinc_log_level_t level, const char *format, ...) {
 #define UTF8
 
 void kinc_log_args(kinc_log_level_t level, const char *format, va_list args) {
+#ifdef KINC_ANDROID
+	va_list args_android_copy;
+	va_copy(args_android_copy, args);
+	switch (level) {
+	case KINC_LOG_LEVEL_INFO:
+		__android_log_vprint(ANDROID_LOG_INFO, "Kinc", format, args_android_copy);
+		break;
+	case KINC_LOG_LEVEL_WARNING:
+		__android_log_vprint(ANDROID_LOG_WARN, "Kinc", format, args_android_copy);
+		break;
+	case KINC_LOG_LEVEL_ERROR:
+		__android_log_vprint(ANDROID_LOG_ERROR, "Kinc", format, args_android_copy);
+		break;
+	}
+	va_end(args_android_copy);
+#endif
 #ifdef KINC_MICROSOFT
 #ifdef UTF8
 	wchar_t buffer[4096];
@@ -111,20 +127,6 @@ void kinc_log_args(kinc_log_level_t level, const char *format, va_list args) {
 	vsnprintf(buffer, 4090, format, args);
 	strcat(buffer, "\n");
 	fprintf(level == KINC_LOG_LEVEL_INFO ? stdout : stderr, "%s", buffer);
-#endif
-
-#ifdef KINC_ANDROID
-	switch (level) {
-	case KINC_LOG_LEVEL_INFO:
-		__android_log_vprint(ANDROID_LOG_INFO, "Kinc", format, args);
-		break;
-	case KINC_LOG_LEVEL_WARNING:
-		__android_log_vprint(ANDROID_LOG_WARN, "Kinc", format, args);
-		break;
-	case KINC_LOG_LEVEL_ERROR:
-		__android_log_vprint(ANDROID_LOG_ERROR, "Kinc", format, args);
-		break;
-	}
 #endif
 }
 

